openapi.yaml

```diff
@@ -249,6 +249,17 @@ paths:
           schema:
             type: string
             format: binary
+          headers:
+            ETag:
+              type: string
+              description: UUID of log entry
```

Comment:
maybe be a bit more verbose here (and in other description strings below) "UUID of log entry made for TSR"